codalab/rest/interpret.py

```diff
@@ -260,16 +261,22 @@ def fetch_interpreted_worksheet(uuid):
     elif directive:
         # Only expand the search item corresponding to the given directive.
         # Used in async loading to only load a single table.
-        item_idx = 0
+        new_items = []
         for i, item in enumerate(worksheet_info['items']):
             (bundle_info, subworksheet_info, value_obj, item_type, id, sort_key) = item
             if directive == formatting.tokens_to_string(value_obj):
                 search_results = perform_search_query(value_obj)
-                item_idx = i
+                new_items.append(item)
```

Comment:
I'm kind of confused by the name new_items, what does it actually include? Maybe change to a clearer name or add a comment saying what's inside?